drv/vsc7448/src/lib.rs

```diff
@@ -366,11 +376,7 @@ impl<'a, R: Vsc7448Rw> Vsc7448<'a, R> {
     ///
     /// Takes the REFCLK frequency, as well as an optional frequency for
     /// REFCLK2 (used to configure the PLL boost).
```

Comment:
This part of the comment should presumably move to `new()`?

Reply:
Yup, done